Tests/SwiftDocCTests/Infrastructure/DataAssetManagerTests.swift

```diff
@@ -196,12 +224,19 @@ class DataAssetManagerTests: XCTestCase {
         var manager = DataAssetManager()
         
         // Test that we build the fuzzy index
-        let images = ["image@2x.png", "assets/woof.JPG"].compactMap(URL.init(string:))
+        let images = [
+          "image@2x.png",
+          "assets/woof.JPG",
+        ].compactMap(URL.init(string:))
         try manager.register(data: images)
         XCTAssertEqual(manager.fuzzyKeyIndex.keys.sorted().map({"\($0)"}), ["image", "woof"])
 
         // Test we include only unique keys in the fuzzy index
-        let imagesWithDuplicates = ["image@2x.png", "assets/woof.JPG", "woof.png"].compactMap(URL.init(string:))
+        let imagesWithDuplicates = [
+          "image@2x.png",
```

Comment:
To be consistent with the rest of the code, we should use a 4-space indent for these

Reply:
Thank you for your kind comment. ☺️